samples/analyze.v1p2beta1.js

```diff
@@ -0,0 +1,383 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+function analyzeTextGCS(gcsUri) {
+  //[START video_detect_text_gcs_beta]
+  // Imports the Google Cloud Video Intelligence library
+  const video = require('@google-cloud/video-intelligence').v1p2beta1;
+
+  // Creates a client
+  const client = new video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const gcsUri = 'GCS URI of the video to analyze, e.g. gs://my-bucket/my-video.mp4';
+
+  const request = {
+    inputUri: gcsUri,
+    features: ['TEXT_DETECTION'],
+  };
+  // Detects text in a video
+  client
```

Comment:
With the change to an async function, we can say:

```js
const [operation] = await client.annotateVideo(request);
const results = await operation.promise();
const textAnnotations = results[0].annotationResults[0].textAnnotations;
// etc....
```

Reply:
Thanks for the help. Updated the code. The only thing I wasn't too sure about was the error catching. Let me know if there's anything additional to be done.